ignite3/src/main/java/site/ycsb/db/ignite3/IgniteJdbcClient.java

```diff
@@ -215,4 +221,33 @@ public void cleanup() throws DBException {
 
     super.cleanup();
   }
+
+  /**
+   * If parameter 'txops' > 0,
+   * then turn off auto commit until we accumulate required quantity of operations,
+   * else leave default auto commit value (true).
+   *
+   * @param operation Operation that produces a result.
+   * @return operation call result.
+   */
+  @Override
+  protected <R> R wrapWithTx(Callable<R> operation) throws Exception {
+    if (!isWrapOpsToTx) {
+      return operation.call();
+    } else {
+      if (CONN.get().getAutoCommit()) {
+        CONN.get().setAutoCommit(false);
+      }
+
+      R result = operation.call();
```

Comment:
What would happen if this call throws a `SQLException`? Should we perform rollback?